tests/unittests/QuantizationTest.cpp

```diff
@@ -117,11 +117,13 @@ TEST(Quantization, Serialize) {
   testSerialization(expected);
 }
 
+#if LLVM_VERSION_MAJOR < 8
```

Comment:
Is this a use case that we really want to test? If it doesn't work on LLVM8 then maybe we don't really care about this use case and can simply delete it?

Reply:
I'm just not sure if this is a bug or a valid behavior: https://github.com/pytorch/glow/issues/2588

Reply:
Okay.